crates/core/src/client/client_connection.rs

```diff
@@ -244,12 +261,20 @@ impl ClientConnection {
         args: ReducerArgs,
         request_id: RequestId,
         timer: Instant,
+        flags: CallReducerFlags,
     ) -> Result<ReducerCallResult, ReducerCallError> {
+        let caller = match flags {
+            CallReducerFlags::FullUpdate => Some(self.sender()),
+            // Setting `sender = None` causes `eval_updates` to skip sending to the caller
+            // as it has no access to the caller other than by id/addr.
+            CallReducerFlags::NoSuccessNotify => None,
```

Comment:
Does this also affect the `sender: Identity` and client address passed to the reducer in WASM? I feel like those should still be passed.

Reply:
Excellent, thanks!